### ISSUE TITLE
Pagination - add prop for page size menu toggle label

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-react#1882]
The consumer should be able to specify the text string for the toggle that displays the page size menu.

We should also include a default string that clearly describes the button. 

NOTE: these updates are expected to be completed after or as part of the update to refactor the Pagination component to use the OptionsMenu component. As part of this update, `button.pf-c-options-menu__toggle` would not be expected to have the attribute `aria-labelledby`.

**Updates for this issue:**
- Include a prop named `ariaLabelPageSize` that, when specified, provides the text string for a value for `aria-label` on `button.pf-c-options-menu__toggle-button`
- ~Define a default string "Items per page" to use when this prop is not specified by the user.~ This was completed in https://github.com/patternfly/patternfly-react/pull/2904

**Comment by jgiardino** [comment]
@mcarrano @stacymcauliffe 

For the page size toggle, does "Page size" work? Or "Items per page"? 

When announced, the whole button gets announced as "[label] button collapsed".

I also need to create an issue for this in core, btw.

**Comment by mcarrano** [comment]
"Items per page" IMO. Page size is a bit ambiguous.

**Comment by jgiardino** [comment]
Thanks! That sounds good to me. I updated the comment description to use that as a text string but kept the prop name as "ariaLabelPageSize" for brevity.

**Comment by dgutride** [comment]
can this one be closed @jgiardino ?

**Comment by jgiardino** [comment]
I don't see these updates in the component, can we keep it open? 

Currently, we include an `aria-label` attribute on the toggle with a static string of "Select". The consumer should be able to provide their own localized string for this. 

Also, the default value for this text string should be "Items per page", which is a more specific label.

**Comment by jessiehuff** [comment]
This seems like a fairly straightforward issue, just giving flexibility to change the `aria-label` of the pagination toggle (which would likely help with localization as well).

From what I understand, we just need to add a prop named ariaLabelPageSize that, when specified, provides the text string for a value for aria-label on button.pf-c-options-menu__toggle-button

**Comment by tlabaj** [comment]
The prop is there it is not documented.
